src/App.tsx

```diff
@@ -1,30 +1,33 @@
-import { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
+import { Film } from './types/Film';
 import data from './api/movies.json';
 
-interface State {
-  movies: Movie[];
-}
+export const App: React.FC = () => {
+  const [movies, addData] = useState<Film[]>(data);
```

Comment:
```suggestion
  const [movies, setMovies] = useState<Film[]>(data);
```
зазвичай називають так: `set` + назва зміної